ts/modules/networkgraph/layouts.ts

```diff
@@ -206,6 +211,23 @@ extend(
 
             this.approximation = options.approximation;
         },
+        update: function (
+            this: Highcharts.NetworkgraphLayout,
+            options: Highcharts.NetworkgraphLayoutAlgorithmOptions,
+            redraw?: boolean
+        ): void {
+            this.options = merge(this.options, options);
+            if (pick(redraw, true) && this.chart) {
+                this.chart.redraw();
+            }
```

Comment:
I don't see any use of this method - is method necessary?
Also, do we need `chart.redraw()` here? Layout up to now was independent from the chart instance.

Reply:
yeah, Right now the method is not used anywhere but in general it works so it is possible to use it. I think we can keep it as it is not too long method. `chart.redraw` might be called outside of the function so it is not needed to be in here.

Reply:
I'm not sure if this generally works - for example updating max number of iterations, or force-functions. Simulation must be reset in order to apply all changes. It seems that only `options.enableSimulation` and `options.theta` options were directly used - so updating `this.options` affects them :)

Reply:
we can use `layout.init` for updating all the other layout parameters. However, after deeper investigation I believe we don't need this method for now. I will add it to the code when necessary.